src/sap/fhir/model/r4/FHIRFilterOperator.js

```diff
@@ -41,7 +41,49 @@ sap.ui.define(["sap/ui/model/FilterOperator"], function(FilterOperator) {
 		 *
 		 * @public
 		 */
-		AP: "ap"
+		AP: "ap",
+		/**
+		 * The set is empty or not (value is false or true)
+		 *
+		 * @public
+		 */
+		PR: "pr",
+		/**
+		 * if a (implied) date period in the set overlaps with the implied period in the value
```

Comment:
[code-style] use starting upper case letters for description of enums

Reply:
done